src/notifiers/telegram-notifier.js

```diff
@@ -0,0 +1,24 @@
+import {logger} from '../logger.js';
+import {TelegramBotApiClient} from '../telegram-bot-api-client.js';
+import {config} from "../config.js";
+
+const telegramBotClient = new TelegramBotApiClient(config.telegram.botURL, config.telegram.botToken);
```

Comment:
I think we want to use the `config` passed from the constructor, so we don't need to import `config.js` again here

Reply:
makes sense, removing...